### ISSUE TITLE
Provide a postgresql by default

### ISSUE THREAD

**Issue by yodatak** [WDaan/hoppscotch-helm#8]
Hi thank for you work it could interersting to have by default a postgresql database like in the docker-compose what do you think ?

**Comment by WDaan** [comment]
Yes! that would make sense. Ofcourse I would include a disclaimer that for production usecases you might want to rely on an external (managed) database instead.
